packages/i18n-calypso/cli/index.js

```diff
@@ -0,0 +1,118 @@
+/** @format */
```

Comment:
Perhaps this can be dropped soon with the work on #28504 #28583 #28842, etc.

Let's not spend a lot of time on CLI cleanup.